simulatorUser/use.rst

```diff
@@ -38,36 +39,64 @@ The set of fibers may be:
 * ``even``: every other fiber, starting with the second.
 * ``all``: all fibers.
 
-The targets may be:
+``DRP_INSTDATA_DIR``
+~~~~~~~~~~~~~~~~~~~~
 
-* ``Arc``: an arc spectrum with all lamps for all fibers.
-* ``Ne``: an arc spectrum with the Ne lamp for all fibers.
-* ``Hg``: an arc spectrum with the Hg lamp for all fibers.
-* ``Flat``: a quartz spectrum for all fibers.
-* ``Comb``: a set of emission lines of equal spacing for all fibers.
-* ``Const``: a constant :math:`F_nu` spectrum for all fibers.
-* ``Sky``: a pure sky spectrum for all fibers.
-* ``Obj``: a combination of sky, flux calibration sources and objects.
+Also before running the simulator the envionment variable ``DRP_INSTDATA_DIR`` needs to be set to point
+to a recent version of the ``drp_instdata`` git repository. For example::
+
+    cd /path/to
+    git clone https://github.com/Subaru-PFS/drp_instdata.git
+    export DRP_INSTDATA_DIR=/path/to/drp_instdata
+
+Location of input spectra
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main simulator command, ``makeSim``, requires the corresponding object spectra
+as input. More specifically, the location of those input spectra.
+While this can be specified through the ``--spectraDir`` option, an alternative
+is to use the default value (which is the current working directory)
+and could create a symbolic link to your spectral files. 
+For example, if your input data are the ``lowz_COSMOS`` data 
+from the ``drp_instdata`` repository, you could create a link as follows::
+
+    ln -s /path/to/drp_instdata/data/objects/lowz_COSMOS/ 1
+where the ``1`` corresponds to the catalogue identifer of the source (eg the value of 
+the ``scienceCatId`` in the ``makePfsDesign`` example shown above).
+
+
+Single image
+------------
+
+The ``makeSim`` script runs the Simulator to produce a single image
+and the associated ``pfsConfig`` file.
+An example command-line is::
+
+    makeSim --detector r1 --pfsConfig --pfsDesignId 1 --exptime 0 --domeClosed --expId 0 --imagetyp BIAS
```

Comment:
`--pfsConfig` isn't explained below.
I think `--expId` has been replaced with `--visit`.

Reply:
I've explained ``--pfsConfig`` now. 
I've not switched from ``--expId`` to ``--visit`` as your changes are not currently merged to master.